### PR TITLE
appease rubocop 0.48

### PR DESCRIPTION
**Pull request by dunn**
Should let #1649 pass.

### CODE REVIEW

**Comment by dunn** [comment]
No idea what's going on with Travis; everything's passing locally.

File: .rubocop.yml [hunk 1]
```diff
@@ -24,6 +31,9 @@ Metrics/LineLength:
 Style/StringLiterals:
   Enabled: false
 
+Style/SymbolArray:
```
Comment (dunn) [inline .rubocop.yml]: I added this to minimize the diff; if we want to use `i%()` I'm happy to.

Comment (jcoyne) [inline .rubocop.yml]: I think this is a good choice.

File: .rubocop.yml [hunk 2]
```diff
@@ -6,13 +6,20 @@ inherit_from:
...
+    - "db/**/*"
+    - "lib/generators/blacklight/templates/**/*"
+    - "spec/**/*"
 
 Rails:
   Enabled: true
 
+# TODO: remove when
+# https://github.com/bbatsov/rubocop/commit/246bef5e1cccd04045a71eee2476cde6c0782e92
+# is part fo a stable release
```
Comment (jcoyne) [inline .rubocop.yml]: 😢

Comment (dunn) [inline .rubocop.yml]: should i fix the typo

**Comment by jcoyne** [comment]
The tests appear to show the spellcheck stuff is failing.  Yet, I don't see any code changes around that.  Do you think Solr 6.5 broke the suggest endpoint?

**Comment by dunn** [comment]
@jcoyne I made some changes to `solr/response/spelling.rb` then moved them to a separate branch and force-pushed them off this PR, so it might have those cached somehow?

**Comment by dunn** [comment]
Gonna re-open on a clean branch.

**Pull request closed without merging**